ui/scripts/package.sh

```diff
@@ -13,12 +13,13 @@ rm -rf $ROOT_DIR/target/${ZIP_FILE_NAME}*.zip
 npm install
 bower install
 
-pids=$(pgrep Xvfb)
```

Comment:
If Xvfb is not running, the exit code of the pgrep will be 1, and therefore the exit code of the variable assignment is also 1. Since "set -e" is set earlier in the commit, the script would always exit as this point.